packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -449,6 +449,159 @@ curl -v -X GET \
+| limit     | Specifies the number of results per page (maximum 200)                                                           | Query      | Number   | FALSE    | 20      |
+| sortBy    | Sorts the tokens by `profile.email`, `profile.serialNumber`, `activated`, `user.id`, `created`, `status`, or `lastVerified` | Query      | String   | FALSE    |        |
+| sortOrder | Specifies the sort order, either `ASC` or `DESC`                                                                 | Query      | String   | FALSE    |         |
+| forDownload | Returns tokens in a CSV for download instead of in the response.  Defaults `limit` to 1000                     | Query      | Boolean  | FALSE    | false   |
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/org/factors/yubikey_token/tokens"
+```
+
+##### Response example
+
+```json
+[
+    {
+        "id": "ykkwcx13nrDq8g4oy0g3",
+        "created": "2020-01-14T21:53:09.000Z",
+        "lastVerified": "2020-01-14T21:53:06.000Z",
+        "lastUpdated": "2020-01-14T21:53:09.000Z",
+        "status": "UNASSIGNED",
+        "profile": {
+            "serial": "000003632071"
+        },
+        "_links": {
+            "self": {
+                "href": "http://${yourOktaDomain}/api/v1/org/factors/yubikey_token/tokens/ykkwcx13nrDq8g4oy0g3",
+                "hints": {
+                    "allow": [
+                        "GET",
+                        "DELETE"
+                    ]
+                }
+            }
+        }
+    },
+    {
+        "id": "ykkxdtCA1fKVxyu6R0g3",
+        "created": "2020-06-09T23:42:05.000Z",
+        "activated": "2020-06-09T23:47:29.000Z",
+        "lastVerified": "2020-06-09T23:47:29.000Z",
+        "lastUpdated": "2020-06-09T23:47:29.000Z",
+        "status": "ACTIVE",
+        "profile": {
+            "serialNumber": "000009508427"
+        },
+        "_links": {
+            "self": {
+                "href": "https://${yourOktaDomain}/api/v1/org/factors/yubikey_token/tokens/ykkxdtCA1fKVxyu6R0g3",
+                "hints": {
+                    "allow": [
+                        "GET"
+                    ]
+                }
+            },
+            "user": {
+                "href": "https://${yourOktaDomain}/api/v1/users/00uu0x8sxTr9HcHOo0g3",
+                "hints": {
+                    "allow": [
+                        "GET"
+                    ]
+                }
+            },
+            "deactivate": {
+                "href": "https://${yourOktaDomain}/api/v1/users/00uu0x8sxTr9HcHOo0g3/factors/ykfxduQAhl89YyPrV0g3",
+                "hints": {
+                    "allow": [
+                        "DELETE"
+                    ]
+                }
+            }
+        }
+    }
+]
+```
+
+### Get a Single YubiKey OTP Token
+
```

Comment:
Can we add this as well between the header and description?

`<ApiOperation method="get" url="/api/v1/org/factors/yubikey_token/tokens/${tokenId}" />`